common/api-review/util.api.md

```diff
@@ -221,6 +223,11 @@ export const getDefaultEmulatorHost: (productName: string) => string | undefined
 // @public
 export const getDefaultEmulatorHostnameAndPort: (productName: string) => [hostname: string, port: number] | undefined;
 
+// Warning: (ae-missing-release-tag) "getDefaults" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
```

Comment:
Was this added after the file was generated? We shouldn't directly modify this file, the tag needs to be added in the comment in the source code (see comment below).